docs/examples/values/prod/bitbucket.yaml

```diff
@@ -0,0 +1,349 @@
+# -- The initial number of pods that should be started at deployment of Bitbucket.
+replicaCount: 1
+
+image:
+  repository: atlassian/bitbucket-server
+  pullPolicy: IfNotPresent
+  # -- The docker image tag to be used. Defaults to the Chart appVersion.
+  tag: ""
+
+serviceAccount:
+  # -- Specifies the name of the ServiceAccount to be used by the pods.
+  # If not specified, but the "serviceAccount.create" flag is set, then the ServiceAccount name will be auto-generated,
+  # otherwise the 'default' ServiceAccount will be used.
+  name:
+  # -- true if a ServiceAccount should be created, or false if it already exists
+  create: true
+  # -- The list of image pull secrets that should be added to the created ServiceAccount
+  imagePullSecrets: []
+  clusterRole:
+    # -- Specifies the name of the ClusterRole that will be created if the "serviceAccount.clusterRole.create" flag is set.
+    # If not specified, a name will be auto-generated.
+    name:
+    # -- true if a ClusterRole should be created, or false if it already exists
+    create: true
+  clusterRoleBinding:
+    # -- Specifies the name of the ClusterRoleBinding that will be created if the "serviceAccount.clusterRoleBinding.create" flag is set
+    # If not specified, a name will be auto-generated.
+    name:
+    # -- true if a ClusterRoleBinding should be created, or false if it already exists
+    create: true
+
+database:
+  # -- The JDBC URL of the database to be used by Bitbucket, e.g. jdbc:postgresql://host:port/database
+  # If not specified, then it will need to be provided via the browser during initial startup.
+  url: jdbc:postgresql://bitbucket-postgres-2.c8hzuxrqaglj.eu-west-1.rds.amazonaws.com/bitbucket
```

Comment:
I think we should make this more generic:
```
url: jdbc:postgresql://postgres.k8s.com:5432/bitbucket
```
We could potentially add also examples for other DBs so people don't need to go to documentation. The other examples can be just commented out.